File: VB/NET40/WhatIsMyBrowser.CommonTypesVB/VersionCheck.vb
Imports Newtonsoft.Json

Public Class VersionCheck
    <JsonProperty("is_checkable")>
    Public Property IsCheckable As Boolean

    <JsonProperty("is_up_to_date")>
    Public Property IsUpToDate As Boolean

    <JsonProperty("latest_version")>
    Public Property LatestVersion As String()

    <JsonProperty("download_url")>
    Public Property DownloadUrl As Uri

    <JsonProperty("update_url")>
    Public Property UpdateUrl As Uri

    <JsonProperty("release_date")>
    Public Property ReleaseDate As DateTimeOffset
End Class

File: CS/NET40/WhatIsMyBrowser.CommonTypes/VersionCheck.cs
using System;
using Newtonsoft.Json;

namespace WhatIsMyBrowser.CommonTypes
{
    public class VersionCheck
    {
        [JsonProperty("is_checkable")]
        public bool IsCheckable { get; set; }

        [JsonProperty("is_up_to_date")]
        public bool IsUpToDate { get; set; }

        [JsonProperty("latest_version")]
        public string[] LatestVersion { get; set; }

        [JsonProperty("download_url")]
        public Uri DownloadUrl { get; set; }

        [JsonProperty("update_url")]
        public Uri UpdateUrl { get; set; }

        [JsonProperty("release_date")]
        public DateTimeOffset ReleaseDate { get; set; }
    }
}